packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/routing.js

```diff
@@ -16,6 +16,8 @@ export default /* @ngInject */ ($stateProvider) => {
           },
         ),
       server: /* @ngInject */ (node) => node,
+      statePrefix: /* @ngInject */ () =>
+        'nutanix.dashboard.nodes.node.general-info',
```

Comment:
no need to add `/* @ngInject */` here as you're defining a constant
```suggestion
      hidePublicBandwidth: () => true,
```